pylintrc

```diff
@@ -575,5 +575,4 @@ min-public-methods=2
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "BaseException, Exception".
-overgeneral-exceptions=BaseException,
-                       Exception
+overgeneral-exceptions=builtins.BaseException
```

Comment:
Did you mean to omit Exception?

Reply:
Yeah, because that's what's causing the pylint issue in https://github.com/rcsb/py-rcsb_utils_seq/blob/master/rcsb/utils/seq/UniProtUtils.py#L637

Reply:
Might be good to consider using a more specific exception type, but not really a high priority for that one line.

Reply:
Right, I don't feel like we need to worry about using too general of an exception, since we really just care about raising some exception. I can see the value in being explicit when possible, but don't think it should be a total blocker for the package.